src/components/stats/Charts/Generic/LpiSingleStatChart.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <div class="total-container">
+        <span class="projects-count">{{ displayedNumber }}</span>
+        <span class="projects-word">{{ $t('stats.projects') }}</span>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'LpiSingleStatChart',
+    props: {
+        number: {
+            type: Number,
+            default: null,
+        },
+    },
+    data() {
+        return {
+            displayedNumber: 0,
+            interval: false,
+        }
+    },
+    mounted() {
+        clearInterval(this.interval)
+        if (this.chartData === this.displayedNumber) return
+        this.interval = window.setInterval(() => {
+            const change = (this.number - this.displayedNumber) / 10
+            this.displayedNumber += change > 0 ? Math.ceil(change) : Math.floor(change)
+        }, 20)
+    },
```

Comment:
il faut ajouter un lifecyle hook "beforeUnmount" avec `window.clearInterval(this.interval)` pour éviter les fuite de mémoire